Lib/gftools/builder/file.py

```diff
@@ -47,6 +47,12 @@ def is_designspace(self):
     def is_font_source(self):
         return self.is_glyphs or self.is_ufo or self.is_designspace
 
+    @cached_property
+    def is_variable(self) -> bool:
+        return (self.is_glyphs and len(self.gsfont.masters) > 1) or (
```

Comment:
This may be too simple since I have a seen a family that has 4 masters and 4 instances which are not MM compatible so it could only export 4 static fonts.

Something like `self.is_glyphs and len(self.gsfont.masters) < len(self.gsfont.instances)` but it still isn't 100% reliable.

Reply:
perhaps `maybe_variable` is a better name for the method. But I think the gftools config provides an even stronger hint as to whether a font is supposed to be variable: if the font dev asks to build a variable font, then it must be variable; if it disables building the VF (is it `buildVariable: false`?) then one should treat it as not variable.

Reply:
> But I think the gftools config provides an even stronger hint as to whether a font is supposed to be variable: if the font dev asks to build a variable font, then it must be variable; if they disables building the VF (is it buildVariable: false?) then one should treat it as not variable.

I'm not sure I understand this. `buildStatic`, `buildVariable`, `buildOTF`, `buildWebfont` etc. are parameters about *targets*, not about *sources*. "I don't want you to build a variable font today" does not mean "this is not a variable font".